src/main/java/org/apache/ibatis/jdbc/ScriptRunner.java

```diff
@@ -186,11 +186,12 @@ private void checkForMissingLineTerminator(StringBuilder command) {
 
   private StringBuilder handleLine(StringBuilder command, String line) throws SQLException, UnsupportedEncodingException {
     String trimmedLine = line.trim();
-    if (trimmedLine.toUpperCase().startsWith("DELIMITER")) {
-        delimiter = trimmedLine.substring(10).trim();
-        return command;
-    }
```

Comment:
I don't know if they worked before, but if the worked, I think this change will broke MySQL scripts, as MySQL uses syntax: 
DELIMITER //
CREATE TRIGGER .... );
END//
DELIMITER ;

Is there a reason why this code was removed?